docs/package.json

```diff
@@ -64,6 +65,7 @@
     "markdown-to-jsx": "^7.1.9",
     "marked": "^4.2.12",
     "moment": "^2.29.4",
+    "moment-hijri": "^2.1.2",
```

Comment:
It was already possible to use this package in docs due to it being in the monorepo scope, because of being included in `x-date-pickers`, but that's a bit error-prone. 🙈